src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -234,8 +240,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
     }
 
     private static LocalDateTime parseTime(String str) {
-        def pbsDatePattern = DateTimeFormatter.ofPattern("EEE MMM ppd HH:mm:ss yyyy").withLocale(Locale.ENGLISH)
-        return LocalDateTime.parse(str, pbsDatePattern)
+        return catchExceptionAndLog {Instant.ofEpochSecond(Long.valueOf(str)).atZone(ZoneId.systemDefault()).toLocalDateTime()}
```

Comment:
Just as remark (no request for change): I know this kind of functions with the "with..." conventions. E.g. 
```
withCaughtAndLoggedExceptions {
   ...
}
```

`catchAndLogExceptions` would be slightly less bumpy :-)